libvast/vast/concept/printable/vast/json.hpp

```diff
@@ -165,7 +168,7 @@ struct json_printer : printer<json_printer<TreePolicy, Indent, Padding>> {
       while (begin != end) {
         if (!indent())
           return false;
-        if (!(*this)(begin->first))
+        if (!(*this) (begin->first))
```

Comment:
clang-format 12 disagrees with CI for these. 🤷‍♂️

Reply:
11 looks better.

Reply:
I agree, hence the last commit.